runfile/trafficgen/trafficgen-remotehosts-runfile.json

```diff
@@ -0,0 +1,107 @@
+{
+  "tags": {
+    "topology": "RHEL10-ctx7-test-002pct"
+  },
+  "endpoints": [
+    {
+      "type": "remotehosts",
+      "settings": {
+        "user": "root"
+      },
+      "remotes": [
+        {
+          "engines": [
+            { "role": "client", "ids": [ 1 ] }
+          ],
+          "config": {
+            "host": "client-host.com",
+            "settings": {
+              "cpu-partitioning": true,
+	      "userenv": "alma8",
+	      "osruntime": "chroot"
+            }
+          }
+        },
+        {
+          "engines": [
+            { "role": "server", "ids": [ 1 ] }
+          ],
+          "config": {
+            "host": "server-host.com",
+            "settings": {
+              "cpu-partitioning": true,
+	      "userenv": "rhel95",
+	      "osruntime": "chroot"
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "run-params": {
+    "num-samples": 10,
+    "max-sample-failures": 3,
```

Comment:
Let's reduce this to some more sane defaults, 10 is a bit extreme for an example...